cmd/controller/main.go

```diff
@@ -2,42 +2,41 @@ package main
 
 import (
 	"crypto/rand"
-	"crypto/rsa"
 	"crypto/x509"
-	"crypto/x509/pkix"
 	goflag "flag"
 	"fmt"
 	"io"
 	"io/ioutil"
 	"log"
-	"math/big"
 	"os"
 	"os/signal"
 	"strings"
 	"syscall"
 	"time"
 
 	flag "github.com/spf13/pflag"
-	"k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
-	certUtil "k8s.io/client-go/util/cert"
 
 	sealedsecrets "github.com/bitnami-labs/sealed-secrets/pkg/client/clientset/versioned"
 	ssinformers "github.com/bitnami-labs/sealed-secrets/pkg/client/informers/externalversions"
 )
 
 var (
-	keyName      = flag.String("key-name", "sealed-secrets-key", "Name of Secret containing public/private key.")
-	keySize      = flag.Int("key-size", 4096, "Size of encryption key.")
-	validFor     = flag.Duration("key-ttl", 10*365*24*time.Hour, "Duration that certificate is valid for.")
-	myCN         = flag.String("my-cn", "", "CN to use in generated certificate.")
-	printVersion = flag.Bool("version", false, "Print version information and exit")
+	keyLabelName    = flag.String("key-label", "sealed-secrets-key", "Label used to identify public/private key pairs in k8s.")
```

Comment:
I _think_ `keyLabelName` is ignored by the code, which is good since I was going to suggest we remove it and just use a hard-coded value :stuck_out_tongue: 
I think we should remove this and just use a hardcoded value.

If we want to support multiple sealed-secrets controllers in the same k8s namespace, then I think we should just have a flag for arbitrary additional label selectors.